src/src/Commands/MQTT.h

```diff
@@ -16,7 +16,14 @@ String Command_MQTT_messageDelay(struct EventStruct *event,
 String Command_MQTT_Publish(struct EventStruct *event,
                             const char         *Line);
 
-#endif // ifdef USES_MQTT
 
 
+
+//MFD: adding subscribe command
+extern void setIntervalTimerOverride(unsigned long id, unsigned long msecFromNow); //defined in Scheduler.ino
```

Comment:
Please move this forward declaration to `ESPEasy_fwdecl.h`
That's the big TODO list of functions now defined in .ino files and needed from .h/.cpp files.

Reply:
ok